dsc/cli.py

```diff
@@ -85,6 +88,62 @@ def reconcile(ctx: click.Context) -> None:
         ctx.exit(1)
 
 
+@main.command()
+@click.pass_context
+@click.option("--sync-data/--no-sync-data", default=False)
+@click.option(
+    "-s",
+    "--source",
+    help=(
+        "Source directory formatted as a local filesystem path or "
+        "an S3 URI in s3://bucket/prefix form"
+    ),
+)
+@click.option(
+    "-d",
+    "--destination",
+    help=(
+        "Destination directory formatted as a local filesystem path or "
+        "an S3 URI in s3://bucket/prefix form"
+    ),
+)
+@click.option(
+    "--dry-run",
+    is_flag=True,
+    help=(
+        "Display the operations that would be performed using the "
+        "sync command without actually running them"
+    ),
+)
+@click.option(
+    "-e",
+    "--email-recipients",
+    help="The recipients of the submission results email as a comma-delimited string",
+    default=None,
+)
+def create(
+    ctx: click.Context,
+    source: str | None = None,
+    destination: str | None = None,
+    email_recipients: str | None = None,
+    *,
+    dry_run: bool = False,
+    sync_data: bool = False,
+) -> None:
+    """Create a batch of item submissions."""
+    workflow = ctx.obj["workflow"]
+
+    if sync_data:
+        ctx.invoke(sync, source, destination, dry_run=dry_run)
```

Comment:
Have we tried this locally?  I'm a little concerned that `sys.exit(return_code)` line in the `sync` CLI command may terminate the parent python process and therefore not make it past this step?

Reply:
Additionally and related, what happens if the `sync` subcommand has an error?  how do we know and how is it handled?